auth-table-based-service/src/main/java/io/stargate/auth/table/AuthTableBasedService.java

```diff
@@ -52,11 +54,16 @@ public void setPersistence(Persistence persistence) {
     this.persistence = persistence;
     this.dataStore = persistence.newDataStore(null, null);
 
-    initAuthTable(this.dataStore);
+    if (Boolean.parseBoolean(System.getProperty("stargate.auth_tablebased_init", "true"))) {
```

Comment:
Maybe move this up to a static field for visibility (like the other configuration)?

Reply:
Sure. I was just doing it here since it's only used in the one place.